src/modules/user/dto/create-wishlist.dto.ts

```diff
@@ -0,0 +1,4 @@
+import { Schema } from 'mongoose';
+export class CreateWishlistDTO {
+  cId?: Schema.Types.ObjectId;
```

Comment:
If someone is creating wishlist, then it should be required.

Reply:
Actually This dto is not at all necessary.

Reply:
We need to destructure the object
Either we have to use @Req method or the dto
no other way to destructure an object of type schemta.types.objectid

Reply:
otherwise the cid will not go and it will become an array of objects instead of an array of objectids